src/getCoinCombination.test.js

```diff
@@ -3,8 +3,61 @@
 describe('getCoinCombination', () => {
   const { getCoinCombination } = require('./getCoinCombination');
 
-  it(`should be declared`, () => {
-    expect(getCoinCombination)
-      .toBeInstanceOf(Function);
+  it('should be declared', () => {
+    expect(getCoinCombination).toBeInstanceOf(Function);
+  });
+
+  it('should return [0, 0, 0, 0] if cents === 0', () => {
+    const res = getCoinCombination(0);
+
+    expect(res).toStrictEqual([0, 0, 0, 0]);
+  });
+
+  it('should return [2, 0, 0, 0] if cents is less than 5', () => {
```

Comment:
Test description is incorrect. It should be 'should return [2, 0, 0, 0] for cents equal to 2'